src/pages/server-sitemap.xml/index.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { getServerSideSitemap } from 'next-sitemap'
+import getNavigation from 'utils/getNavigation'
+
+const DOMAIN_URL = 'https://developers.vtex.com/docs/api-reference'
+
+function getEndpoint(element: any) {
+  if (element.type === 'openapi') {
+    const url: any = {}
+    url.loc = `${DOMAIN_URL}/${element.slug}/#${element.method.toLowerCase()}-${
+      element.endpoint
```

Comment:
Tem q pegar o endpoint e trocar os `{` e os `}` por `-`
```suggestion
      element.endpoint.replaceAll('{', '-').replaceAll('}', '-') 
```